org-formation/300-account-defaults/organization-tasks.yaml

```diff
@@ -59,3 +59,20 @@ SceptreLambdaBucket:
     IncludeMasterAccount: true
     Account: '*'
     Region: us-east-1
+
+ItKmsKey:
+  Type: update-stacks
+  Template: ./kms-key.yaml
+  StackName: it-kms-key
+  MaxConcurrentStacks: 10
+  Parameters:
+    AliasName: alias/itkey
+    AdminPrincipalArns:
+      - !Sub 'arn:aws:iam::${AWS::AccountId}:root'
+      - !Sub 'arn:aws:iam::${AWS::AccountId}:role/OrganizationFormationBuildAccessRole'
```

Comment:
Include `arn:aws:iam::${AWS::AccountId}:role/AccountName-admin` in this list? Is the `AccountName` a stack output?

Reply:
actually users in the JC `aws-admins` group can assume `OrganizationFormationBuildAccessRole` role,  so i think it’s good as is.